src/main/java/classes/Farm.java

```diff
@@ -4,7 +4,7 @@
  * use it for free
  */
 
-package clasess;
+package classes;
```

Comment:
split classes into several folders. As we discussed you can split it into service classes and instances classes.